detox/android/detox/src/main/java/com/wix/detox/DetoxManager.java

```diff
@@ -102,6 +101,14 @@ public void onAction(final String type, final String params, final long messageI
         handler.post(new Runnable() {
             @Override
             public void run() {
+
+                final ExternalAction externalAction = externalActions.get(type);
```

Comment:
What does External mean?

Reply:
actions that are externally invoked (by the test runner).

Reply:
Do you mean `device` API?

Reply:
hmm not necessarily, i think